libs/langgraph/langgraph/pregel/remote.py

```diff
@@ -357,17 +389,24 @@ def stream(
     ) -> Iterator[Union[dict[str, Any], Any]]:
         merged_config = merge_configs(self.config, config)
         sanitized_config = self._sanitize_config(merged_config)
+        updated_stream_modes, include_updates = self._get_stream_modes(stream_mode)
 
         for chunk in self.sync_client.runs.stream(
             thread_id=sanitized_config["configurable"]["thread_id"],
             assistant_id=self.graph_id,
             input=input,
             config=sanitized_config,
-            stream_mode=stream_mode,  # type: ignore
+            stream_mode=updated_stream_modes,
             interrupt_before=interrupt_before,  # type: ignore
             interrupt_after=interrupt_after,  # type: ignore
             stream_subgraphs=subgraphs,
         ):
+            if chunk.event == "updates":
+                if INTERRUPT in chunk.data:
+                    raise GraphInterrupt()
+                if not include_updates:
+                    continue
+
             yield chunk
```

Comment:
I think this needs to unnest the tuple w stream_mode, payload if the caller had asked for a single stream mode

Reply:
Clarifying my understanding:
1. `RunsClient.stream()` returns `Iterator[StreamPart]`.
1. `StreamPart` is a `NamedTuple` containing `event` and `data`.
1. If a client calls `RunsClient.stream()` with only 1 stream mode (or no stream mode, which means the default is `values`), then only `data` is returned. Is this correct?

If this is correct, should we make the change in the SDK or the API instead?